packages/airnode-validator/src/ois/ois.ts

```diff
@@ -67,7 +67,16 @@ export const reservedParameterSchema = z
     default: z.string().optional(),
     fixed: z.string().optional(),
   })
-  .strict();
+  .strict()
+  .refine((value) => {
```

Comment:
What's the difference between `refine` and `superRefine`? Why do we use `superRefine` everywhere else?

Reply:
refine is a shorthand for super refine, but it's less flexible. One disadvantage is that it doesn't set the `path` to the offending code. For simple validations though, refine is less boilerplate.